### ISSUE TITLE
Why is the Curriculum PDF 12 megabytes?

### ISSUE THREAD

**Issue by surfer190** [cncf/curriculum#35]
I find it is weird that a three page PDF is 12.2Mb.

Surely you can optimise it and save people a bit of bandwidth when downloading.

**Comment by dankohn** [comment]
Good point. @taylorwaggoner could you please see what is going on here?

**Comment by michaelhaycroft** [comment]
Would be better if it was provided in markdown or something a little more consumable. Cannot copy-paste syllabus items from the pdf files using e.g. Chrome... I have to manually type them into a list.
